docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::note
+The documentation is written with [docusaurus](https://docusaurus.io/). You don't need install anything specific for docusaurus but it can be helpful to reference their documentation if needed.
+:::
+
+:::note
+If you are working with the documentation from within VSCode+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll n eed to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+In order to view and verify documentation updates locally you'll need to install `node.js` and `npm`. Please see [nodejs.org](https://nodejs.org/) for details on how to install these tools.
+
+## Verifying Documentation Updates Locally
+
+To verify documentation updates locally, follow the following procedure.
+
+1. Navigate to the `docs` folder
+2. Run `npm ci` to build the necessary tools if you haven't run it before or the zmk sources were updated
```

Comment:
I have run into issues on Windows where `npm ci` wouldn't correctly install things while `npm install` would. It's worth checking if that's still a problem and maybe making a note of it here if it is.

Reply:
I've never needed to run npm install. Should I still include a note/info block?

Reply:
If `npm ci` is working now, then no need to include it. We can always update it later if that issue pops back up.